pgbot/commands/__init__.py

```diff
@@ -97,6 +97,18 @@ async def handle(invoke_msg: discord.Message, response_msg: discord.Message = No
     ):
         return
 
+    async with db.DiscordDB("bot_mutes") as db_obj:
+        muted_users = db_obj.get({})
+
+    if invoke_msg.author.id in muted_users:
+        await embed_utils.send_2(
```

Comment:
You need to edit `response_message`, rather than sending a new one. Also, this block can prolly go a bit below.
(Minor nitpick, please fix the super long string that is well exceeding 88 chars in a line here ;) )